go/multi_user_pub.go

```diff
@@ -0,0 +1,2 @@
+channel := client.Channels.Get("doc:mars-launch-plan")
+channel.Publish(context.Background(), "update", map[string]string{ "pos": "charAt", "insert": "Send Grimes first" })
```

Comment:
Please split this over two lines, and make `charAt` a variable reference rather than a string